### PR TITLE
Add pannenkoek

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pannenkoek/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - qiime
+    - rpy2
+    - numpy
+    - matplotlib
+    - argparse
```
Comment (bgruening) [inline recipes/pannenkoek/meta.yaml]: Is argparse really needed? I guess it is annotated in the setup.py, but unless we do want to support python2.6 this should be removed from here and the setup.py

Comment (druvus) [inline recipes/pannenkoek/meta.yaml]: Good catch. I will fix it.

Comment (bgruening) [inline recipes/pannenkoek/meta.yaml]: I created a PR upstream, but maybe this can be patched out in the meantime?

**Comment by druvus** [comment]
@bgruening seems that pannenkoeken is now maintained as koeken. Closing until this old version is needed.

**Pull request closed without merging**